firefox/manifest.json

```diff
@@ -40,9 +40,17 @@
   "commands": {
     "_execute_action": {
       "suggested_key": {
-        "default": "Ctrl+Shift+L",
-        "mac": "Command+Shift+L"
-      }
+        "default": "Ctrl+Alt+K",
+        "mac": "Command+Alt+K"
+      },
+      "description": "Open the Kagi extension"
+    },
+    "summarize-active-page": {
+      "suggested_key": {
+        "default": "Ctrl+Alt+S",
+        "mac": "Command+Alt+S"
+      },
+      "description": "Summarize the currently active page"
```

Comment:
I changed the shortcut in Firefox because it allows `Alt` instead of `Shift`, and that way the `K` can be the trigger to open. As for summarizing, the `S` made a lot of sense (couldn't do it for Chrome, because `Ctrl+Shift+S` is taken, and `Alt` can't be used as a secondary modifier).